src/sources/kubernetes_logs/mod.rs

```diff
@@ -123,50 +153,69 @@ pub struct Config {
 
     /// Ignore files with a data modification date older than the specified number of seconds.
     #[serde(default)]
+    #[configurable(metadata(docs::type_unit = "seconds"))]
     ignore_older_secs: Option<u64>,
 
     /// Max amount of bytes to read from a single file before switching over
     /// to the next file.
+    ///
     /// This allows distributing the reads more or less evenly across
     /// the files.
+    #[configurable(metadata(docs::type_unit = "bytes"))]
     max_read_bytes: usize,
 
-    /// The maximum number of bytes a line can contain before being discarded. This protects
-    /// against malformed lines or tailing incorrect files.
+    /// The maximum number of bytes a line can contain before being discarded.
+    ///
+    /// This protects against malformed lines or tailing incorrect files.
+    #[configurable(metadata(docs::type_unit = "bytes"))]
     max_line_bytes: usize,
 
-    /// How many first lines in a file are used for fingerprinting.
+    /// The number of lines to read when generating a unique fingerprint of a log file.
+    ///
+    /// This is helpful when some containers share common first log lines.
+    ///
+    /// **WARNING**: If the file has less than this amount of lines then it won't be read at all.
+    /// This is important since container logs are broken up into several files, so the greater
+    /// `lines` value is, the greater the chance of it not reading the last file/logs of
+    /// the container.
+    #[configurable(metadata(docs::type_unit = "lines"))]
     fingerprint_lines: usize,
```

Comment:
same, default lines

Reply:
https://deploy-preview-15724--vector-project.netlify.app/docs/reference/configuration/sources/kubernetes_logs/#fingerprint_lines the default is rendered?